documentation/assemblies/assembly-kafka-connect.adoc

```diff
@@ -32,13 +33,14 @@ m¦FileStreamSinkConnector
 
 |===
 
-In {ProductName}, you can use the Cluster Operator to deploy a Kafka Connect to your Kubernetes cluster.
-OpenShift also supports Kafka Connect Source-2-Image (S2I) clusters.
+You can use the Cluster Operator to deploy a Kafka Connect cluster containing the built-in connectors to your Kubernetes cluster. Deploying a Kafka Connect Source-2-Image (S2I) cluster is supported on OpenShift only. For more information, see xref:using-openshift-s2i-create-image-str[].
```

Comment:
This doesn't really make sense if you don't already know what a "Kafka Connect Source-2-Image cluster" is. It also might be interpreted to mean that only the built-in connectors are supported on non-OpenShift. 

I think we should first explain that to use other connector classes you need to one of these two procedures (one of which is openshift only) to prepare images, and then explain that the CO is able to use the images thus created to deploy a KC cluster.

Reply:
I've rewritten this based on your comments. Please can you check it?

> To use other connector classes, you need to prepare connector images by following one of these procedures:
> 
> * xref:creating-new-image-from-base-{context}[] 
> 
> * xref:using-openshift-s2i-create-image-str[] (OpenShift only)
> 
> The Cluster Operator can use images that you create to deploy a Kafka Connect cluster to your Kubernetes cluster.
>